libraries/AP_HAL_ChibiOS/hwdef/MicoAir405v2/defaults.parm

```diff
@@ -0,0 +1,10 @@
+#BATTERY MONITOR ON
+BATT_MONITOR 4
+
+#Serial Port defaults
+SERIAL1_PROTOCOL 2
+SERIAL2_PROTOCOL 42
```

Comment:
it is possible to enable two OSD types...since you have on board AND a DJI connector and have defaulted its serial connection to DisplayPort, I suggest you add OSD_TYPE2=5, so it is plug- and -play with HD goggles

Reply:
Got it. Thanks for your suggestions! I will test it in the board.